dbt/include/bigquery/macros/adapters/apply_grants.sql

```diff
@@ -0,0 +1,29 @@
+{% macro  bigquery__get_show_grant_sql(relation) %}
+    {% if not target.location %}
+        {{ exceptions.raise_compiler_error("In order to use the grants feature, you must specify a location ") }}
+    {% endif %}
+
+    select privilege_type, grantee from {{ relation.project }}.`region-{{ target.location }}`.INFORMATION_SCHEMA.OBJECT_PRIVILEGES
```

Comment:
Learning something new everyday -- I didn't know about the information schema for [object privileges being region-specific for BigQuery](https://cloud.google.com/bigquery/docs/information-schema-object-privileges#scope_and_syntax).

An idea (which might be a bad one!):
- We could move your region-specific logic into the [BigQueryInformationSchema class](https://github.com/dbt-labs/dbt-bigquery/blob/6828a619b4ce2a88bc01770ec5e227e916c3e1d3/dbt/adapters/bigquery/relation.py#L52-L53)

That way, any user could refer to this view in a more generalized way like this and it will magically handle the region-specific bits:
`{{ relation.information_schema("object_privileges") }}`

(Not sure if I got the casing right -- maybe "object_privileges" needs to be upper-cased.)

Reply:
This ended up being included in this PR!